source/language/gates.rst

```diff
@@ -341,7 +341,7 @@ respectively.
    qubit[5] a;
    qubit f;
    reset f;
-   negctrl(2) @ ctrl(3) @ x a, f;
+   negctrl(2) @ ctrl(3) @ x a[0], a[1], a[2], a[3], a[4], f;
```

Comment:
A minor point, but #325 will fix this, so best to keep it out of this PR to avoid conflicts.